codex-ui/src/vue/card/Card.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div :class="[$style.card, orientation === 'horizontal' ? $style['card--horizontal'] : '']">
+    <div :class="$style['card__cover']"></div>
+    <div :class="$style['card__footer']">
+      <Heading :level="3">{{ title }}</Heading>
+      <div :class="[$style['card__updatedAt'], 'text-ui-subtle']">{{ updatedAt }}</div>
+    </div>
+    <slot></slot>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps } from 'vue';
+import Heading from '../heading/Heading.vue';
+import type { CardOrientation } from './Card.types';
+
+/**
+ * NoteListItem props, receive Note
+ *
+ * @param {string} title - Note title
+ * @param {string} updatedAt - Last updated date of note
+ * @param {string} buttonText - Button text
+ * @param orientation - Orientation of card
+ */
+defineProps<{
+  title: string;
+  updatedAt: string;
+  orientation: CardOrientation;
+}>();
+</script>
+
+<style module lang="postcss">
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 222px;
```

Comment:
better to use a separate local CSS variable for this value
for example, `--card-width: 222px;`